src/Settings/ChangePassword/ChangePasswordScreen.tsx

```diff
@@ -1,31 +1,29 @@
 import {useNavigation} from '@react-navigation/native'
 import React from 'react'
 import {defineMessages, useIntl} from 'react-intl'
-import {ScrollView, StyleSheet, View} from 'react-native'
+import {ScrollView, StyleSheet, TextInput as RNTextInput, View} from 'react-native'
 import {SafeAreaView} from 'react-native-safe-area-context'
 import {MutationOptions, useMutation} from 'react-query'
 
-import {Button, TextInput} from '../../../legacy/components/UiKit'
-import {Checkmark} from '../../../legacy/components/UiKit/TextInput'
 import {errorMessages} from '../../../legacy/i18n/global-messages'
 import {COLORS} from '../../../legacy/styles/config'
 import {REQUIRED_PASSWORD_LENGTH, validatePassword} from '../../../legacy/utils/validators'
-import {Spacer} from '../../components'
+import {Button, Checkmark, Spacer, TextInput} from '../../components'
 import {useSelectedWallet} from '../../SelectedWallet'
 import {WalletInterface} from '../../types'
 
 export const ChangePasswordScreen = () => {
   const strings = useStrings()
   const navigation = useNavigation()
 
-  const currentPasswordRef = React.useRef<{focus: () => void} | null>(null)
+  const currentPasswordRef = React.useRef<RNTextInput>(null)
```

Comment:
react native paper issue (will not fix)
https://github.com/callstack/react-native-paper/issues/1322#issuecomment-699166622